lib/Driver/ToolChains.cpp

```diff
@@ -1286,6 +1300,24 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   if (!Linker.empty()) {
     Arguments.push_back(context.Args.MakeArgString("-fuse-ld=" + Linker));
   }
+  
+  // Configure the toolchain
+  // By default, use the system clang++ to link.
+  const char * Clang = "clang++";
```

Comment:
Reasonable enough.  Swift already requires clang.  Perhaps this should be an option then?  So if the clang driver is named something other than clang, it gets the right name.